en/docs/learn/ai/agent/inline-agent-introduction.md

```diff
@@ -0,0 +1,75 @@
+# Introduction to inline agents
+
+In this tutorial, you'll learn how to connect an AI agent to a GraphQL service, enabling the agent to be invoked directly within a GraphQL resolver. This demonstrates the use of an **inline agent**—a powerful capability in the Ballerina Integrator.
+
+Unlike [chat agents](/learn/ai/agent/chat-agent-introduction), which are exposed as REST APIs for external interaction, **inline agents** are not tied to an API endpoint. Instead, they can be invoked programmatically from anywhere within your integration logic, just like a regular function call.
+
+In this example, we'll define a GraphQL schema with a query that invokes the inline agent to generate dynamic responses based on input parameters. The agent runs within the resolver logic and returns results directly as part of the GraphQL response.
+
+## Prerequisites
+
+- Sign up at [OpenAI](https://platform.openai.com/signup/).
+- Get an API key from the [API section](https://platform.openai.com/docs/api-reference/authentication).
+
+### Step 1: Create a new integration project
+
+1. Click on the **Ballerina Integrator** icon in the sidebar.
+2. Click on the **Create New Integration** button.
+3. Enter the project name as `GraphqlService`.
+4. Select the project directory by clicking on the **Select Location** button.
+5. Click the **Create New Integration** button to generate the integration project.
+
+    <a href="{{base_path}}/assets/img/inline-agent-introduction/inline-agent-step1.gif"><img src="{{base_path}}/assets/img/inline-agent-introduction/inline-agent-step1.gif" alt="Create a new integration project" width="70%"></a>
```

Comment:
Images are not rendering. Check the paths. Also make sure images are in matchign path structure to the left navigation. 

For example : This image/gif should goes to a folder like below.

`en/docs/assets/img/learn/ai/agent/inline-agents`

Check and fix paths of all images by moving images to correct folder

Reply:
I have doubled checked it images are rendering properly. To view them locally need to updated the `base_path` to `http://localhost:8000` in `mkdocs.yml` file. Will rename the images.

Reply:
fixed with [Update gif, filenames and titles of learn pages](https://github.com/wso2/docs-bi/pull/75/commits/d85e0aeb2cb5787de2bcaea0fb988afbd7ccf6d9)